### DISCUSSION TITLE
Influx DB missing.

### DISCUSSION THREAD

**Discussion by dinucacid** [David00/rpi-power-monitor#125] [Q&A]
Hi,
I've set up the power monitor on a Pi4 4Gb and used the prebuilt OS image. The Pi boots with no errors, connects to wifi and I can log into Grafana and see the dashboards. I have the sensors installed and I assume they are working.
My problem is the dashboards have no data and it appears that the Influx db called power_monitor is not setup.
I logged into Influx on the command line and tried running some of the sample queries on the database info page in the documentation and it came back with the error that the db was missing.

The power monitor is at a remote location which I usually visit on weekends, so this may be a long process. David has given me a couple of commands to run when I'm there next so I'll report back after that.

Thanks

Andrew

**Comment by dinucacid** [comment]
Hi,
Further to this, I have imaged another SD card on a different computer and used that. 
This seemed to work better. I can see an influx db called power_monitor but there are no measurements coming into it.
I checked that the power monitor service was active and also the influx db.
The error about the missing db is gone from Grafana but there is still no data.

I checked the sensor output and got 10.45 V AC at 773.5 Hz. I put a clamp meter around the wire and saw 1.5 amps.
![IMG_3379](https://github.com/David00/rpi-power-monitor/assets/11545425/b64855b2-3dcc-4c41-900b-86b073b2c215)
![IMG_3377](https://github.com/David00/rpi-power-monitor/assets/11545425/63552353-fbaa-470e-b4cb-d1385741a0cb)

Any suggestions of what I check next ?

Thanks

Andrew

**Comment by David00** [comment]
After conversing via email with @dinucacid, it appears this issue was due to setting a custom username via Raspberry Pi Imager when flashing the Custom OS Image to the microSD card.  I will look at improving the docs to include a notice in the instructions.
